benchmark/mnist/mnist.yaml

```diff
@@ -34,7 +34,7 @@ training:
     maximum_keys_in_memory: 1000
     storage_backend: "database"
     limit: -1
-    reset_after_trigger: True
+    tail_triggers: 0
 data:
```

Comment:
The functions of `reset_after_trigger` and `tail_triggers` overlap, that's why previously we have to do validations like when `reset_after_trigger == True` then `tail_trigger` cannot be bigger than 0.

The function of `reset_after_trigger` can completely be replaced by `tail_triggers`. If and only if `tail_triggers == 0`, `reset_after_trigger == True`.

Therefore I remove `reset_after_trigger` for clarity. It becomes a field in the config internally computed based on the value of `tail_triggers`.

Reply:
Thank you, that seems reasonable! I suppose internally having this field still makes sense because it allows for some optimization when computing the TTS, in comparison to using a general tail triggers integer.